youtubeExtension/output.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Settings</title>
+  </head>
+  <body>
+    <h1 class="header">SETTINGS</h1>
+
+    <div class="page-flex">
+
+      <!--FLEX ITEM ONE FOP TOGGLES-->
+      <div class="toggle-box">
+        <div class="title-row">
+          <h3>TOGGLES</h3>
+        </div>
+        <div class="row">
```

Comment:
This is fine, but it's another place where you have all the toggles hardcoded. This means that if you ever want to change the toggles (like, letting the user customize their toggle options), you'll need to change this code as well as a bunch of other places where you define all the toggle options.

If you were coding this up from scratch, you'd probably want to define a list of toggle options in one single place (or really, one place on the frontend and possibly a second place on the backend). Then you'd loop through that list of toggles everywhere you need to, like right here. That would make it *much* easier to let users customize the toggle options, because you'd only need to change that one single function to return the correct values, and everything else would update automatically.